external/libfetch/fetch.h

```diff
@@ -90,7 +90,9 @@ struct url_ent {
 #define	FETCH_URL	18
 #define	FETCH_VERBOSE	19
 
-__BEGIN_DECLS
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
I am puzzled here, nothing call by c++ should include that as we don't call the c++ command at all and cc should be fine with it